test/repositories/AuxiliaryChainRepository/save.test.ts

```diff
@@ -102,7 +102,7 @@ describe('AuxiliaryChainRepository::save', (): void => {
     Util.assertAuxiliaryChainAttributes(updatedAuxiliaryChain, auxiliaryChain);
   });
 
-  it('should fail when ostGatewayAddress is null', async (): Promise<void> => {
+  it('should fail when eip20GatewayAddress is null', async (): Promise<void> => {
     const auxiliaryChain = new AuxiliaryChain(
       chainId,
       originChainName,
```

Comment:
this file has local variables like `ostGatewayAddress` & `ostCoGatewayAddress`. please change these too.